src/std/text/csv.ss

```diff
@@ -310,13 +310,18 @@
            (string-any (cut w/opt char-needs-quoting? <>) x))
        #t))
 
-;; Given a list of LINES, each of them a list of fields, and a PORT,
+;; Given a list or lsit of LINES, each of them a list of fields, and a PORT,
 ;; format those lines as CSV according to the current syntax parameters.
 (def/opt (write-csv-lines lines port)
-  (for-each (cut w/opt write-csv-line <> port) lines))
+  (def normal-table (or (null? lines)
```

Comment:
This function needs a better name than that, and to be outside this definition.

I'm still not convinced it's a good idea.

Reply:
I'll moved it out and gave it another name.

When working, I sometimes have to "process" some csv files to match various values or filter files. Most of the time I just fire up a REPL to do the work. It is not uncommon that the result is a list-of-values (strings), that I then want to turn back into a csv and send it someone. Right now when using `(write-csv-file "out.csv" ["a" "b"])` the procedure returns a strage error. Instead of catching the error and saying that the input should be a list-of-lines, I thought, why not automatically turn it into the right result.